### PR TITLE
Restore logging configurations

### PR DESCRIPTION
**Pull request by RassK**
Restoring `FileLogEnabled`, `StdoutLogEnabled`, `StdoutLogTemplate` to be sync with main.

### CODE REVIEW

**Comment by pellared** [comment]
Update of the `USAGE.md` is missing.

**Comment by RassK** [comment]
> Update of the `USAGE.md` is missing.

Somehow got confused that they are already there 😬

File: docs/USAGE.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ Use these environment variables to configure the tracing library:
 | `SIGNALFX_TAGS` | Comma-separated list of key-value pairs to specify global span tags. For example: `"key1:val1,key2:val2"` |  |
 | `SIGNALFX_LOGS_INJECTION` | Enable to inject trace IDs, span IDs, service name and environment into logs. This requires a compatible logger or manual configuration. | `false` |
 | `SIGNALFX_EXPORTER` | The exporter to be used. The Tracer uses it to encode and dispatch traces. Available values are: `DatadogAgent`, `Zipkin`. | `Zipkin` |
+| `SIGNALFX_STDOUT_LOG_ENABLED` | `false` | Enables `stdout` logging. This is disabled by default. |
+| `SIGNALFX_STDOUT_LOG_TEMPLATE` | `"[{Level:u3}] {Message:lj} {NewLine}{Exception}{NewLine}"` | Configures `stdout` log template. |
```
Comment (pellared) [inline docs/USAGE.md:57]: can you add some information regarding the template formatting?

**Review by pellared — APPROVED** [review]

File: docs/USAGE.md (hunk 1 above)
Comment (RassK) [reply docs/USAGE.md]: Updated.

File: docs/USAGE.md [hunk 2]
```diff
@@ -53,6 +53,9 @@ Use these environment variables to configure the tracing library:
 | `SIGNALFX_TAGS` | Comma-separated list of key-value pairs to specify global span tags. For example: `"key1:val1,key2:val2"` |  |
 | `SIGNALFX_LOGS_INJECTION` | Enable to inject trace IDs, span IDs, service name and environment into logs. This requires a compatible logger or manual configuration. | `false` |
 | `SIGNALFX_EXPORTER` | The exporter to be used. The Tracer uses it to encode and dispatch traces. Available values are: `DatadogAgent`, `Zipkin`. | `Zipkin` |
+| `SIGNALFX_STDOUT_LOG_ENABLED` | `false` | Enables `stdout` logging. This is disabled by default. |
```
Comment (Kielek) [inline docs/USAGE.md:56]: description then default value?

Comment (RassK) [reply docs/USAGE.md]: that's awkward... didn't even notice that main and next-ver has different col oder. :flushed:

**Review by pellared — APPROVED** [review]

**Review by Kielek — APPROVED** [review]

**Pull request merged**